draft-irtf-cfrg-hpke.md

```diff
@@ -1567,32 +1611,62 @@ HPKE is not tolerant of lost messages. Applications MUST be able to detect when
 a message has been lost.  When an unrecoverable loss is detected, the application MUST discard
 any associated HPKE context.
 
-### Non-Goals
+### Downgrade Prevention
+
+HPKE assumes that the sender and recipient agree on what algorithms to use.
+Depending on how these algorithms are negotiated, it may be possible for an
+intermediary to force the two parties to use suboptimal algorithms.
+
+### Replay Protection
+
+The requirement that ciphertexts be presented to the `ContextR.Open()` function
+in the same order they were generated by `ContextS.Seal()` provides a degree of
+replay protection within a stream of ciphertexts resulting from a given context.
+HPKE provides no other replay protection.
+
+### Forward Secrecy
 
-HPKE does not provide several features that a more high-level protocol might
-provide, for example:
+HPKE ciphertexts are not forward secret with respect to recipient compromise
+in any mode. This is because no ephemeral randomness but only long-term secrets
+are used on the side of the recipient.
```

Comment:
```suggestion
in any mode. This means that compromise of long-term recipient secrets allows
an attacker to decrypt past ciphertexts encrypted under said secret. This is because 
only long-term secrets are used on the side of the recipient.
```